common/source/docs/common-smartap-pdb.rst

```diff
@@ -0,0 +1,73 @@
+.. _common-smartap-pdb:
+
+===========================================
+Sky-Drones SmartAP Power Distribution Board
+===========================================
+
+Overview
+========
+
+`SmartAP PDB <https://www.mauch-electronic.com/#webs-bin-563c137b7add4cb229080f55>`__ (Power Distribution Board) is a board which allows transferring the power from 
+the battery to ESCs / Motors and generate power supply for the flight controller and 
+other peripherals with different voltage levels. Also, PDB provides the functionality for battery 
+voltage / current measurements. SmartAP PDB makes high-power lines connections easier and much more reliable.
+
+
+.. image:: ../../../images/smartap-pdb/smartap-pdb-top-side.jpg
+    :target: ../_images/smartap-pdb/smartap-pdb-top-side.jpg
+
+Specifications
+==============
+
+- Size: 65x65 mm, four 3mm mounting holes
+- Input voltage up to 60 Volts (14S)
+- Capability to handle extremenly high currents (peak current up to 400A)
+- Power input from the main battery, possibility to connect up to 4 independent batteries
+- 12 pairs of pads (6 on top, 6 on bottom) for powering up to 12 motors (all possible airframe configurations supported)
+- Integrated voltage and current sensors
+- Integrated DC-DC converter from 10-60 V input (up to 14S battery) to 5V output to power peripherals
+- Integrated DC-DC converter from 10-60 V input (up to 14S battery) to 12V output to power peripherals
+- 5V and 12V power output terminals (standard 2.54mm/0.1" connectors)
+- Integrated loud electromagnetic sounder (buzzer)
+- Power output for the flight controller (both 5V and battery VIN)
+- Fully compatible with SmartAP, APM and PX4 autopilots
+
+Size and Weight
+===============
+
+- Length: 65mm
+- Width: 65mm
+- Height: 14mm
+- Weight: 16g
+
+
+Buy
+===
+
+Sky-Drones store - purchase `here <https://sky-drones.com/parts/smartap-pdb.html>`__
+
+Pinouts
+=======
+
+SmartAP Power Distribution Board pinout diagram is shown below. Big pads in the rear side are intended for the main battery connection. Up to four independent batteries can be connected using the thick wires (e.g. 8-10 AWG) to be able to handle high current loads and even more using thinner wires. Both top and bottom, left and right sides have pads for ESCs power supply connection. Therefore, up to 12 ESCs can be connected.
+
+.. image:: ../../../images/smartap-pdb/smartap-pdb-pinout.png
```

Comment:
the use of the labeled pin Vin and what voltage it represents is unclear

Reply:
Added more detailed explanation in the image

Reply:
spelling error....so it was quicker for me to change the image....which I did, squashed to one commit...and will merge...thanks!